tests/e2e/tosa_ops/BUILD.bazel

```diff
@@ -249,11 +249,254 @@ iree_check_single_backend_test_suite(
     target_backend = "vulkan-spirv",
 )
 
+CUDA_SRCS = enforce_glob(
```

Comment:
I know we already have duplication, but this isn't very maintainable and pretty messy to have to duplicate large globs of things - maybe this means using something else besides enforce_glob or a smarter enforce_glob that allows us to use the same source of truth srcs list but only have the excludes differ, etc.

Reply:
Do you think this refactoring could land before changing how test suites are defined? As-is, this increases coverage and removes some CMake-only code.